apps/prairielearn/src/middlewares/selectAndAuthzAssessment.html.ts

```diff
@@ -0,0 +1,39 @@
+import { html } from '@prairielearn/html';
+
+import { HeadContents } from '../components/HeadContents.html.js';
+import { Navbar } from '../components/Navbar.html.js';
+import { config } from '../lib/config.js';
+
+export function AccessDenied({ resLocals }: { resLocals: Record<string, any> }) {
+  const courseInstance = resLocals.course_instance.id;
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${HeadContents({ resLocals })}
+      </head>
+      <body>
+        ${Navbar({ resLocals, navPage: 'assessment_instance' })}
+        <main id="content" class="container">
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Assessment Unavailable</div>
+            <div class="card-body">
+              <p>This assessment's configuration does not allow you to access it right now.</p>
+              <p>
+                This is the intended behavior (and not an error) based on your current time,
```

Comment:
Nit, I really don't think we need to spell this out.

```suggestion
                This is the intended behavior based on your current time,
```